spec/element_spec.rb

```diff
@@ -39,6 +39,14 @@ class PageWithElement < SitePrism::Page
     expect(page).to respond_to :wait_for_some_slow_element
   end
 
+  it 'should be able to wait for an element to not exist' do
+    class PageWithElement < SitePrism::Page
```

Comment:
move this class to the top, and define both css / xpath selector inside one class. If you can do this for all specs, and define just the classes that are required that would be nice.

Then if you can use a let block to instantiate the `page`, so we're just calling `page`